packages/bigquery/src/table.js

```diff
@@ -473,7 +459,7 @@ Table.prototype.createWriteStream = function(metadata) {
  * @throws {Error} If destination format isn't recongized.
  *
  * @example
- * var gcs = gcloud.storage({
+ * var gcs = require('@google-cloud/storage')({
```

Comment:
Assuming that the overview gets moved into the constructor description, why do we need this? Shouldn't the tokenized overview contain this already?